cardano-lib/mainnet-config.nix

```diff
@@ -6,10 +6,12 @@
 {
   ##### Locations #####
 
-  ByronGenesisFile = ./mainnet-byron-genesis.json;
+  ByronGenesisFile = ./mainnet + "/byron-genesis.json";
```

Comment:
why not just `./mainnet/byron-genesis.json`?

Reply:
because I want all the files in the same store path. Main reason is alonzo genesis *may* need to contain a reference to another json file relative to where it is.